### ISSUE TITLE
Rendering Filepicker images

### ISSUE THREAD

**Issue by bcackerman** [mileszs/wicked_pdf#372]
I render filepicker images (ex. `https://www.filepicker.io/api/file/KidMFBsnQ46vZ2E5UnHc+file.jpeg/convert?fit=clip&h=100&w=100`) using `image_tag` like this:

```
= image_tag image.filepicker_url+'+file.'+image.mime_type.split('/')[-1]+'/convert?fit=clip&h=100&w=100', {class: 'invoice-image image-thumb-70-width'}
```

which displays normally in HTML but PDF displays nothing, just a blank.  

I used `wicked_pdf_image_tag` like so:

```
= wicked_pdf_image_tag image.filepicker_url+'+file.'+image.mime_type.split('/')[-1]+'/convert?fit=clip&h=100&w=100', {class: 'invoice-image image-thumb-70-width'}
```

but get `undefined method 'pathname' for nil:NilClass` 

What's going on?  Am I able to use Filepicker to show images in a PDF version of a page?

**Comment by unixmonkey** [comment]
Could you show me the erb code for the Wooden Cotton logo as well as the Seen logo?

**Comment by bcackerman** [comment]
Yeah `= image_tag account.logo_url(:thumb2x), class: "img-responsive"`  I'm using carrierwave where you pass in the size of the image you'd like
